centrapay/centrapay-docs
# Update social media link previews for Farmlands pages

Test plan:

- Share the Farmlands pages on Slack and Notion - see that they appear as expected
- Check that https://socialsharepreview.com/?url=https://docs.centrapay.com/connections/farmlands appears as expected for Facebook, LinkedIn and Twitter
- Check that https://socialsharepreview.com/?url=https://docs.centrapay.com/guides/farmlands-pos-integration appears as expected for Facebook, LinkedIn and Twitter
- Check that the title on https://docs.centrapay.com is "Centrapay Docs"
- Check that every other page has ` - Centrapay Docs` appended to the end of the title
- Regression: Check that the Farmlands landing page still has the banner image
